README.md

```diff
@@ -1,42 +1,42 @@
-# MAAS Frontend
+# maas-ui
 
-This is a mono repo for MAAS frontend projects.
+This is repository contains sourcecode for the [MAAS](http://maas.io) web  frontend.
 
 [![CircleCI](https://circleci.com/gh/canonical-web-and-design/maas-ui/tree/master.svg?style=svg)](https://circleci.com/gh/canonical-web-and-design/maas-ui/tree/master)
 
-# Projects
+It is comprised of the following [yarn workspaces](https://yarnpkg.com/lang/en/docs/workspaces/):
 
-## UI
+  - *legacy*: the angularjs maas client.
+  - *ui*: the new react maas client (all new work should be in this workspace).
+  - *shared*: shared components consumed by both legacy and ui.
```

Comment:
Maybe not mention components here as I think we should use this for any code we can share between the two.